### ISSUE TITLE
Request: Provide Homebrew Package

### ISSUE THREAD

**Issue by tisba** [awslabs/fargatecli#34]
It would be super awesome if you could provide an "official" homebrew tap for fargate.

This way it would be very nice and simple to install fargate on macOS without having to have know anything about how to compile go or downloading things from GitHub releases.

You can do that e.g. by using https://github.com/goreleaser/goreleaser. Many options are available and I'd be happy to help if that's something you'd consider, @jpignata.

**Comment by jpignata** [comment]
100%! Thanks for opening this.

**Comment by dstroot** [comment]
This would be really great.  I manage everything I install via homebrew.

**Comment by chenrui333** [comment]
Homebrew has shipped the formula with v0.3.1, please try it out.

Props to @vicendominguez for initiating the efforts.

**Comment by vicendominguez** [comment]
```
brew install fargatecli
Updating Homebrew...
==> Downloading https://homebrew.bintray.com/bottles/fargatecli-0.3.1.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/7b/7bdf49489c1f9e4ae108e79204c68a3f77145befe1e4584e5f7cf3629ff4f1ec?__gda__=exp=1588273398~hmac=1bb63c0fd669e1121eff331e3bae9
######################################################################## 100.0%
==> Pouring fargatecli-0.3.1.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/fargatecli/0.3.1: 6 files, 24.4MB

$ fargatecli service list --cluster arn:aws:ecs:us-east-1:123456789:cluster/cluster
NAME	IMAGE										CPU	MEMORY	LOAD BALANCER	DESIRED	RUNNING	PENDING
filer	gcr.io/registry-cluster/filer:05eblablablablacef707dbab2acd0ad.nginx	         1024	4096	cluster              1	1	0

```
💯 !!!
